weechat-remote-notify.py

```diff
@@ -111,17 +111,17 @@ def on_msg(*a):
         option = w.config_get("weechat.look.prefix_network")
         if sender == w.config_string(option):
             return w.WEECHAT_RC_OK
-        if data == "private" or highlight == "1":
 
+        if data == "private" or highlight == 1:
             #set buffer
             buffer = "me" if data == "private" else w.buffer_get_string(buffer, "short_name")
 
             #set time - displays message forever on highlight
-            if highlight == "1" and data == "private":
+            if highlight == 1 and data == "private":
```

Comment:
if you want I can put the old comparison in an `or` if this is needed for other weechat versions.